addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6451,22 +6459,22 @@ msgstr ""
 
 #: system/settings/darwin.xml
 msgctxt "#13612"
-msgid "Universal Remote"
+msgid "Universal remote"
 msgstr ""
 
 #: system/settings/darwin.xml
 msgctxt "#13613"
-msgid "Multi Remote (Harmony)"
+msgid "Harmony remote"
 msgstr ""
 
 #empty strings from id 13614 to 13619
 
 msgctxt "#13620"
-msgid "Apple Remote Error"
+msgid "Apple remote error"
```

Comment:
Under protest will change

Apple is a brand, remote is a device. Apple Remote is marketing spew.